build.proj

```diff
@@ -77,4 +85,17 @@
     <RemoveDir Directories="$(BinDir)" />
   </Target>
 
+  <PropertyGroup>
+    <UserLocalFolder Condition="'$(OsEnvironment)'!='Unix'">$(LocalAppData)/</UserLocalFolder>
+    <UserLocalFolder Condition="'$(OsEnvironment)'=='Unix'">$(HOME)/.local/share/</UserLocalFolder>
+  </PropertyGroup>
+
+  <Target Name="CleanPackages">
+    <RemoveDir Directories="$(PackagesDir)" />
+  </Target>
+
+  <Target Name="CleanPackagesCache">
+    <RemoveDir Directories="$(UserLocalFolder)NuGet/Cache/;$(UserLocalFolder)NuGet/v3-cache/;$(UserLocalFolder)dnu/cache/" />
+  </Target>
```

Comment:
Yes this is intended to be temporary. As you can tell I just copied what you had in your PR. I just signed off on your buildtools PR. I generally like to stage things by putting them directly in corefx first and then promoting them to buildtools.